### PR TITLE
Fixes for automated build

### PR DESCRIPTION
**Pull request by deschuma**
1. Fix a build break
2. Workaround an issue with using getenv in release builds which resulted in tests failing

### CODE REVIEW

File: src/dcap_provider.cpp [hunk 1]
```diff
@@ -69,12 +69,33 @@ static char PROCESSOR_CRL_NAME[] = "https%3a%2f%2fcertificates.trustedservices.i
...
 {
+#ifdef __LINUX__
     const char* env_value = getenv(env_variable.c_str());
-
     if (env_value == NULL)
     {
         return std::string();
     }
+#else
+    char* env_value = (char*)malloc(MAX_ENV_VAR_LENGTH);
```
Comment (msft-jonnadul) [inline src/dcap_provider.cpp]: Use unique_ptr<string> here so you don't have to worry about freeing it later on.

File: src/logging.cpp [hunk 2]
```diff
@@ -1,5 +1,4 @@
 // Licensed under the MIT License.
-#define _CRT_SECURE_NO_WARNINGS
```
Comment (msft-jonnadul) [inline src/logging.cpp]: Have you checked how this builds in both Linux and Windows?

File: src/dcap_provider.cpp (hunk 1 above)
Comment (deschuma) [reply src/dcap_provider.cpp]: thanks! I've implemented the change

**Comment by msft-jonnadul** [comment]
LGTM, feel free to merge.

**Pull request merged**